examples/cryptocurrency-advanced/backend/src/transactions.rs

```diff
@@ -95,7 +95,7 @@ pub struct CreateWallet {
 }
 
 /// Cryptocurrency service transactions.
-#[service_interface]
+#[service_interface(exonum(dispatcher = "CryptocurrencyService"))]
```

Comment:
Is it necessary to wrap macro arguments into `exonum(..)`? This seems redundant.

Reply:
This is only for uniformity with the `#[derive(ProtobufConvert)]`. 
I suppose that if we don't need to support it, I can rename `service_interface` into `exonum_service` and remove redundant nesting level.